### PR TITLE
add OIDC setup instructions as FAQ in README

### LINKED ISSUE #169: Document how to set up OIDC login with WordPress
Write an FAQ in the README with instructions on how to set up [wp-openid-connect-server](https://github.com/Automattic/wp-openid-connect-server). Also provide pointers on how to set up the homeserver for OIDC.

### PR DESCRIPTION
**Pull request by ashfame**
Fixes #169 

Feel free to tweak it further :)

### CODE REVIEW

**Review by akirk — COMMENTED** [review]
I had some ideas for improving the text, what do you think?

File: README.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ add_filter( 'chatrix_instances', function ( array $default_instances ) {
 } );
 ~~~
 
+### How can I configure WordPress user accounts to register and login on Matrix homeserver?
+The plugin will respect the login methods configured on the Matrix homeserver. You can configure [Open ID Connect (OIDC) login on Synapse](https://matrix-org.github.io/synapse/latest/openid.html). In order to make your WordPress install an OpenID Connect Server/Provider, you can use [OpenID Connect Server plugin](https://wordpress.org/plugins/openid-connect-server/).
```
[the remark this answers is not in the document]
Comment (ashfame) [reply README.md]: Looks great! 👍

File: README.md [hunk 2]
```diff
@@ -57,6 +57,13 @@ add_filter( 'chatrix_instances', function ( array $default_instances ) {
 } );
 ~~~
 
+### How can I configure WordPress user accounts to register and login on Matrix homeserver?
```
[the remark this answers is not in the document]
Comment (ashfame) [reply README.md:60]: Great!

**Comment by ashfame** [comment]
Going to merge this, but feel free to make further edits to the README.

**Pull request merged**